app/src/main/java/org/p2p/wallet/home/ui/select/SelectTokenFragment.kt

```diff
@@ -62,4 +67,32 @@ class SelectTokenFragment : BaseFragment(R.layout.fragment_select_token) {
             emptyTextView.isVisible = isEmpty
         }
     }
+
+    override fun onQueryTextSubmit(query: String?): Boolean = false
+
+    override fun onQueryTextChange(newText: String?): Boolean {
+        val searchText = newText.orEmpty()
```

Comment:
it looks like that all this filtering and checks should be in presenter, it's his responsibility to define what to show on UI